next.config.js

```diff
@@ -1,7 +1,12 @@
+const withNextra = require('nextra')({
```

Comment:
Could we don't use `nextra`? It looks too *professional* and **doesn't match the current memos style**, maybe we can introduce the mdx syntax first.

Reply:
if we just using md/mdx, many time will be spent in designing/layouting/styling the document system, why don't we just use a prefect lib for hosting documents in nextjs?